publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/PermissionStrategy.java

```diff
@@ -47,6 +48,11 @@ protected boolean isVerifiedContributor(Contributor contributor) {
         return contributor.getIdentity() != null && contributor.getIdentity().getId() != null;
     }
 
+
+    protected boolean hasPublishedFiles() {
+        return publication.getAssociatedArtifacts().stream().anyMatch(PublishedFile.class::isInstance);
+    }
```

Comment:
There is a hasUnpublishedFile in CuratorPermissionStrategy. Maybe move that down to PermissionStreategy so they are side-by-side?